projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -232,4 +254,8 @@ export class SearchBoxComponent {
       this.ignoreCloseEvent = false;
     });
   }
+
+  ngOnDestroy() {
```

Comment:
I don't know how this is working, but it needs to implement OnDestroy 

However, that's also a breaking change I believe